src/Core/Datatypes/DenseColumnMatrix.h

```diff
@@ -48,6 +48,12 @@ namespace Datatypes {
 
     explicit DenseColumnMatrixGeneric(size_t nrows = 0) : EigenBase(nrows) {}
 
+    DenseColumnMatrixGeneric(std::vector<T> vals) : EigenBase(vals.size())
```

Comment:
```suggestion
    DenseColumnMatrixGeneric(const std::vector<T>& vals) : EigenBase(vals.size())
```

Reply:
I was using this constructor to easily make matrices for testing. It was useful, but I'm no longer using this class in the tensor code. Do you want me to delete the changes to this file or add the suggestions?

Reply:
Delete away!